Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -13139,10 +13139,14 @@ void WebPageProxy::substitutionsPanelIsShowing(CompletionHandler<void(bool)>&& c
     completionHandler(TextChecker::substitutionsPanelIsShowing());
 }
 
-void WebPageProxy::showCorrectionPanel(AlternativeTextType panelType, const FloatRect& boundingBoxOfReplacedString, const String& replacedString, const String& replacementString, const Vector<String>& alternativeReplacementStrings)
+void WebPageProxy::showCorrectionPanel(AlternativeTextType panelType, const FloatRect& boundingBoxOfReplacedString, const String& replacedString, const String& replacementString, const Vector<String>& alternativeReplacementStrings, std::optional<FrameIdentifier> rootFrameID)
```

Comment:
Change alternativeReplacementStrings from a const Vector<String>& to a Vector<String>&&.  Same with the const String& parameters.